src/Collection/CollectionInterface.php

```diff
@@ -619,6 +619,25 @@ public function first();
      */
     public function last();
 
+    /**
+     * Returns the last N elements of a collection
+     *
+     * ### Examples:
+     *
+     * ```
+     * $items = [1, 2, 3, 4, 5];
+     *
+     * $last = (new Collection($items))->lastN(3);
+     *
+     * // Result will look like this when converted to array
+     * [3, 4, 5];
+     * ```
+     *
+     * @param int $howMany The number of elements at the end of the collection
+     * @return \Cake\Collection\CollectionInterface
+     */
+    public function lastN($howMany);
```

Comment:
Is tailFetch() a better name? :bike:

Reply:
Why isn't it just a param for `last()`?

Reply:
- `last` always returns a single element, not a collection
- `tail` in order languages means everything but the first element. It would be confusing to me, at least
-  I liked `lastN` because it is really close when searching for last or autocompleting.

Reply:
I suggest `takeLast()` as counterpart of `take()`

Reply:
> tail in order languages means everything but the first element.

Guess it's my ignorance but only other popular `tail` I know of is in Unix and it does allow specifying N number of lines to get from the end of file.

Reply:
@ADmad well, I'm not opposed to using `tail` it was just weird for me. I agree that there are also many languages where tail has a different meaning

Reply:
`takeLast` is also a great suggestion @garas

Reply:
:+1: to `takeLast`. I agree that `tail` means a different thing in functional languages and we shouldn't use it for this.